app/components/common/flash_component.html.erb

```diff
@@ -1,8 +1,8 @@
 <% if flash.any? %>
-  <div class="fixed top-20 inset-x-0 m-4 z-50" data-turbo-temporary>
+  <div class="fixed top-20 inset-x-0 m-4 z-50" <%= helpers.temporary_flash_or_empty(flash.all?{|type, _msg| type == 'notice' }) %>>
   <div class="flex flex-col items-end space-y-4">
     <% flash.each do |type, msg| %>
-      <div data-controller="dismissible-alert" data-turbo-temporary class="max-w-sm w-full pointer-events-auto rounded-md <%= type == 'notice' ? "bg-green-50" : "bg-red-50" %> p-4">
+      <div data-controller="<%= type == 'notice' ? 'dismissible-alert' : 'dismissible-alert-no-timeout' %>" <%= helpers.temporary_flash_or_empty(type == 'notice') %> class="max-w-sm w-full pointer-events-auto rounded-md <%= type == 'notice' ? "bg-green-50" : "bg-red-50" %> p-4">
```

Comment:
tuto celkom dobra finta je class_names helper + teda premenoval by som  to na `dismissible`